port/win32/omrintrospect.c

```diff
@@ -176,13 +175,8 @@ suspend_all_preemptive(struct PlatformWalkData *data)
 		passA = !passA;
 	} while (a_count != b_count);
 
-	if (a_count != b_count) {
-		goto error_return;
```

Comment:
Why are you removing this?

Reply:
This condition is always false. The execution flow only gets there if the loop above exits normally.
Additionally, if the condition somehow wasn't false, then there is a chance to get to 'error_return' label having 'result' variable uninitialised.